developers/wallets/exchanges.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: subpage
+title: "Handling a VC API Exchange in a Web Wallet"
+permalink: /developers/wallets/exchanges/
+---
+
+## {{ title }}
+
+> A VC API workflow defines a particular set of steps for exchanging verifiable
+> credentials between two parties across a trust boundary. Each step can involve
+> the issuance, verification, transmission, or presentation of verifiable
+> credentials.
```

Comment:
Can we link to the spec's "Workflows and Exchanges" section here?

Reply:
Done in https://github.com/credential-handler/chapi.io/commit/6c51f3b4c80fe917ad139653b78a0d1b9847b8e8